### ISSUE TITLE
other passwors apps have accessability

### ISSUE THREAD

**Issue by leosemilie** [PhilippC/kp2a_accservice_autofill#20]
other passwors apps have accessability password filling such ass keeper and lastpass for the purpose of passwords and not unabled people so why does this app claim the separation is because google only allows accessibility use by apps for unabled people purpose only

**Comment by PhilippC** [comment]
Google emailed me I should remove the accessibility service. I heard they probably reverted that decision, but they never informed me officially. KP2A now has Android 8 Autofill which works better and is more future safe. I decided to keep accessibility as plugin only.

**Comment by lupalby** [comment]
Unfortunately the Android 8 Autofill doesn't work in most browsers,
including Google's own Chrome. That's the reason why I gave the Autofill
function a good try but I was happy when I found you still had the
accessing service available as a plug-in. I use it all the time even if I'm
on Android 8.
I read that Android P will support Autofill in browsers too, then I'll
definitely switch to the Autofill and uninstall the plug-in.

Anyway, thank you so much for the great app! I use it multiple times a day,
a game changer!
